src/main/python/main.py

```diff
@@ -9,24 +9,31 @@
 from PySide6.QtCore import \
     Qt, QCoreApplication, QTranslator, QLocale, QSettings
 
-from ayab.ayab import GuiMain
-from ayab import utils
+from typing import TYPE_CHECKING, Any, cast
+if TYPE_CHECKING: #TODO: why does mypy not resolve the absolute import correctly?
+    from .ayab.ayab import GuiMain
+    from .ayab import utils
+    cached_property = property #HACK: from https://github.com/python/typing/discussions/1102#discussioncomment-2376328
+else:  
+    from ayab.ayab import GuiMain
+    from ayab import utils
+    from fbs_runtime.application_context.PySide6 import cached_property
 
 
-class AppContext(ApplicationContext):  # 1. Subclass ApplicationContext
+class AppContext(ApplicationContext): #type: ignore # 1. Subclass ApplicationContext
```

Comment:
fbs isn't typed, so the inheritance throws an error. We could get rid of this one if we added some basic typing to fbs_runtime. But that is even more out of scope than a lot of the oos stuff I did here already.